lib/cmds/tasks_cmds/create-foundation-task.js

```diff
@@ -35,8 +35,12 @@ exports.builder = yargs => {
     describe: 'The indexed date.',
     type: 'string'
   }).option('performer-id', {
-    describe: 'The ID of the FHIR Organization resource that performed the seqeuncing.',
+    describe: 'The ID of the FHIR Organization resource that performed the sequencing.',
     type: 'string'
+  }).option('index-type', {
+    describe: 'The index job to perform: all/variant/cnv.',
+    type: 'string',
+    demandOption: false
```

Comment:
You can specify options with:  `choices: ['all', 'variant', 'cnv]`

Reply:
I should have guessed there was a standard way to express that. Update forthcoming.